specs/interop/derivation.md

```diff
@@ -281,6 +273,17 @@ cast keccak $(cast concat-hex 0x000000000000000000000000000000000000000000000000
 # 0xe54b4d06bbcc857f41ae00e89d820339ac5ce0034aac722c817b2873e03a7e68
 ```
 
+### L2 Gas Limit Requirements
+This upgrade requires `3_320_000` in gas. In order to upgrade successfully the following invariant MUST be maintained:
```

Comment:
You'll need to run `just lint-specs-toc` to fix the Table Of Contents

Also, run `just lint-specs-md-fix` to fix the linting (things like whitespace between heading and text)

Reply:
done!